src/caterpillar.jl

```diff
@@ -52,6 +52,23 @@ end
 
 ranefinfo(m::GeneralizedLinearMixedModel, args...; kwargs...) = ranefinfo(m.LMM, args...; kwargs...)
 
+"""
+    ranefinfotable(ri::RanefInfo)
+
+Return the information in `ri` as a column table (`NamedTuple` of `Vector`s) 
+"""
+function ranefinfotable(ri::RanefInfo)
+    cnames, levels = ri.cnames, ri.levels
+    k = length(cnames)
+    l = length(levels)
+    (;
+        cname=repeat(cnames; inner=l),
+        level=repeat(levels; outer=k),
+        μ=vec(ri.ranef),
+        σ=vec(ri.stddev),
```

Comment:
Naming is hard....

Given that these are only means for LMM, is there another name that would be short and sweet?

Or maybe just add a note to the doc string about the fields and their meaning?

Reply:
Do you think `cmode` and `cstddev` would be better than `μ` and `σ`?

The phrase "conditional mode" came from me being pedantic.  Technically "conditional standard deviation" applies under the same circumstances as "conditional mean" - i.e. only to LMMs.  But I wouldn't know what to call it for a GLMM so I sort of punted there. :smiley:

Reply:
I like the `c` vocabulary. It also makes it clear these standard deviations are not the standard deviations from VarCorr.

Reply:
In that case I think I will rename `cname` to `name` to avoid confusion with the `c` prefix.